tests/routers/test_traefik.py

```diff
@@ -36,7 +36,7 @@ def test_create_with_cloudflare_without_credentials(self, update_flag, get_provi
                 'name': 'datapusher',
             },
             'spec': {
-                'default-root-domain': 'default',
```

Comment:
If the purpose of this test is to check that we assert on no credentials, then this fix won't be testing that.

Reply:
Got it